package-lock.json

```diff
@@ -14323,12 +14320,6 @@
         "postcss": "^7.0.14"
       }
     },
-    "icu4c-data": {
-      "version": "0.64.2",
-      "resolved": "https://registry.npmjs.org/icu4c-data/-/icu4c-data-0.64.2.tgz",
-      "integrity": "sha512-BPuTfkRTkplmK1pNrqgyOLJ0qB2UcQ12EotVLwiWh4ErtZR1tEYoRZk/LBLmlDfK5v574/lQYLB4jT9vApBiBQ==",
-      "dev": true
-    },
```

Comment:
```suggestion
    "icu4c-data": {
      "version": "0.64.2",
      "resolved": "https://registry.npmjs.org/icu4c-data/-/icu4c-data-0.64.2.tgz",
      "integrity": "sha512-BPuTfkRTkplmK1pNrqgyOLJ0qB2UcQ12EotVLwiWh4ErtZR1tEYoRZk/LBLmlDfK5v574/lQYLB4jT9vApBiBQ==",
      "dev": true
    },
```

Reply:
I made the changes that you suggested, Saturn :)
Can I ask what I should do to avoid having this issue in future PRs? Should I avoid committing my package-lock.json?

Reply:
Thanks bro! My usual flow is to include changes on `package-lock.json` only when I made changes to `package.json` like when I added or modified dependency. Other than that, I normally ignore changes to `package-lock.json`.

Reply:
Thanks for the tip, pare! I'll keep that in mind for next PR :) 
Stay safe!